### ISSUE TITLE
Variable font instances don't meet Google Fonts requirements

### ISSUE THREAD

**Issue by jenskutilek** [jenskutilek/homecomputer-fonts#21]
The recent changes (added instances and renamed default instances produced those fontbakery errors:

| Name | current | expected |
 | :--- | :--- | :--- | 
| Slight | SCAN=-90.0, BLED=0.0 | N/A | 
| Maximum | SCAN=100.0, BLED=0.0 | N/A |
 | Touching | SCAN=-3.0, BLED=0.0 | N/A |
 | Evenly | SCAN=-65.0, BLED=0.0 | N/A |
 | Delicate | SCAN=-100.0, BLED=0.0 | N/A |
| Overlapping | SCAN=18.0, BLED=0.0 | N/A |
 | Overexposed | SCAN=57.0, BLED=0.0 | N/A |
 | Close | SCAN=-30.0, BLED=0.0 | N/A |
 | Regular | N/A | SCAN=-30.0, BLED=0.0 |

@vv-monsalve can you please advise?

**Comment by vv-monsalve** [comment]
This is reported cause, in regular cases, we don't allow `fvar` instances outside the `wght` and `ital` axes.
However, we could make an exception here, given the font only has custom axes. I was referring to this in the PR that we'll need to discuss internally before having a final approval.

**Comment by vv-monsalve** [comment]
Regarding the "output" values for the SCAN axis that you see there, they were taken by translating the internal positions in the range (20:250) 1:1 to the final axis range (-100:100). 

Now, given this is a custom axis, I think it would be possible to decide to what positions they are mapped, making them more _abstract_, in a similar fashion as `wgth` internal values are mapped to positions happening at fixed 100 intervals.

I'm thinking of something like the A or B options below. What would you say?

However, this also would need to be discussed and approved internally before reaching a final decision.
<br>
<img width="488" alt="Screen Shot 2023-09-06 at 11 52 57" src="https://github.com/jenskutilek/homecomputer-fonts/assets/48698976/e1af2457-14d6-48f4-b78a-4a0eaf3c10e1">

**Comment by davelab6** [comment]
@m4rc1e wdyt about the fvar named instances here? sounds ok to me

**Comment by davelab6** [comment]
@jenskutilek is there a semantic value to the internal units?

**Comment by jenskutilek** [comment]
@davelab6 The internal units are the height of the scan line in font units. A full scan line is 128 units in height, so the default master's value of 100 leaves a small gap between the lines.

**Comment by m4rc1e** [comment]
I agree with Viv. Let's allow custom instances for this family since it only has a Regular on the wght axis. We already do this for Kablammo. Apologies for my late reply as well.

**Comment by vv-monsalve** [comment]
> A full scan line is 128 units in height

Are there any established values for the scan lines? I think Dave's question about semantic values relates to identifying meaningful values for the "output" numbers on the `SCAN` axis.

**Comment by davelab6** [comment]
> Per mille is ok

Viviana reminded me that in the GF axis registry review meeting, we already decided against per-mille because we want to encourage broader re-use of the axis, so we want to go with a percent-from-default relative range.

**Comment by vv-monsalve** [comment]
I made a long reasoning to arrive to the same conclusion. But, with a difference in interpretation. 
> stretching out the negative end and making it travel at about twice the speed 

We would be making the negative end travel more slowly, actually.
Reasoning visually + proposing final mapping intervals:


<img width="779" alt="Screen Shot 2023-10-05 at 12 56 29" src="https://github.com/jenskutilek/homecomputer-fonts/assets/48698976/7e7a1844-e4ad-4f15-8aa2-fc7b22a5c3cc">

**Comment by davelab6** [comment]
I think you've shown visually my math is incorrect, but not provided a correct result :) it's neither -100 nor -53, what is it?

**Comment by davelab6** [comment]
After more review, I don't see a problem with my math, though. 100:150 is 2:3. 80 / (3/2) == 80 * (2/3) == 53.333. I guess the best way to go is to test if you build the VF twice with the internal range and the new -52..100 range, and see if you get the expected same visual output on font gauntlet and font goggles...

**Comment by vv-monsalve** [comment]
I was writing this:

It is correct cause the original interpolation is not linear. So this would be respecting that.

> The per mille values (min/def/max of 20/100/250) are absolute; if we make them relative, they become -80/0/+150.

If we are going to fit the positive range value (150) into 100, that is shrinking it a 66,66%.
Then, to keep the same "shrinking" proportion for the negative range it would be 80 x 66,66% = 53,3

**Comment by vv-monsalve** [comment]
I'm confident about the remapping, I just want us to see what values the intermediate positions would map, and be ok with them.

**Comment by davelab6** [comment]
Oh I see, I misinterpreted your image; I now see your 1st row of red/blue lines has the same proportional balance as the 3rd row, its just a scale difference. This visually confirms the maths :) So, please proceed with -53..100 and @jenskutilek let us know if you feel this is a problem.

**Comment by vv-monsalve** [comment]
Upon further review, the proportional remapping of the range (-80 to -53 and 150 to 100) is ok. However, the intervals displayed in the above plot are not accurate. While I distributed the defined steps evenly across both sections of the range, the values do not progress uniformly in the design.

This is a new graphic with the actual distribution in the design plus two options for the mapping:
- Making it 1:1
- Taking regular intervals (in a similar fashion as it is done for the weight axis)


<img width="719" alt="Screen Shot 2023-10-06 at 9 46 52" src="https://github.com/jenskutilek/homecomputer-fonts/assets/48698976/6f9ff29f-566e-49b2-92ac-2965ebdfecef">

**Comment by jenskutilek** [comment]
@vv-monsalve Thank you for going into this much detail on this :)

I tend towards option 1. The instances are set rather randomly in this font, and I expect the SCAN axis mainly to be used for fine-tuning the appearance. So maybe it is more important to have a linear progression as you move along the axis, and it doesn't matter so much that the distance between the instances is similar.

**Comment by vv-monsalve** [comment]
@jenskutilek I took this case to reason about this custom axis mapping. It was a good test case ;)

> The instances are set rather randomly in this font, and I expect the SCAN axis mainly to be used for fine-tuning the appearance. So maybe it is more important to have a linear progression as you move along the axis, and it doesn't matter so much whether the distance between the instances is similar.

I briefly considered advocating for the second option. However, [after running more tests](https://github.com/vv-monsalve/homecomputer-fonts/issues/1), I realized it was more accurate or informative not to normalize the intervals since the first would not offer the same changes as the others.
